src/sdcard.cpp

```diff
@@ -93,3 +95,21 @@ SDCard::SdCardDirectory SDCard::openDir(const String &pathName)
 
     return result;
 }
+
+SDCard& SDCard::load()
+{
+    if (loadedSDCard_ == nullptr)
```

Comment:
Let's not do this like that.

Passkey is a multicore µC so, this should be implemented as

```.c
SDCard& SDCard::load()
{
    static SDCard sdCardSingelton;
    return sdCardSingelton;
}
```

or we need a mutex.

And managed pointers are clearly prefered over `new` and `delete`.

Reply:
I am getting a LoadProhibited error with EXCVADDR 0x00000000 (so apparently a nullpointer exception) when I attempt this. I have opted for an implementation using a shared_ptr for now.